src/aws/roles.tf

```diff
@@ -231,3 +231,78 @@ resource "aws_iam_role_policy_attachment" "event_rule_lambda_invoke_policy_attac
   role       = "${aws_iam_role.event_rule_role.name}"
   policy_arn = "${aws_iam_policy.event_rule_lambda_invoke_policy.arn}"
 }
+
+# Migrate lambda role
+#####################
+
+resource "aws_iam_role" "lambda_migrate_invocation_role" {
+  name = "${terraform.workspace}-marsha-lambda-migrate-invocation-role"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "lambda_migrate_logging_policy_attachment" {
+  role       = "${aws_iam_role.lambda_migrate_invocation_role.name}"
+  policy_arn = "${aws_iam_policy.lambda_logging_policy.arn}"
+}
+
+resource "aws_iam_policy" "lambda_migrate_s3_access_policy" {
```

Comment:
I named it with `migrate` because this policy will evolve with the lambda policies requirement. Today I only need to list buckets but tomorrow I will probably need more permission on the s3 service and I will modify this policy, I will not create a new one.